### PR TITLE
Kinematic: right_arm->panda_arm

### PR DESCRIPTION
**Pull request by mlautman**
[live](http://tutorials.picknik.ai/doc/kinematic_model/kinematic_model_tutorial.html)

### CODE REVIEW

**Comment by davetcoleman** [comment]
Add note in tutorial that the RobotModel builds off the URDF, and link to http://tutorials.picknik.ai/doc/urdf_srdf/urdf_srdf_tutorial.html

**Comment by davetcoleman** [comment]
In the intro paragraph explain the diff between Model and State... something like

RobotModel contains the relationships between all links and joints including their joint limit properties. It improves URDF by further separating the robot out into planning groups defined in the SRDF (Link)

RobotState contains vectors of each joint's position and optionally velocity and acceleration
RobotState also contains helper functions for setting the arm location based on the end effector location (Cartesian pose) and for computing Cartesian trajectories.

**Comment by davetcoleman** [comment]
Add a picture to this tutorial of the Panda with all the TF links displayed on the arm in Rviz like with http://library.isr.ist.utl.pt/docs/roswiki/attachments/tf/frames2.png

Also explain else where that RobotState as Forward Kinematics built in, partially duplicating TF functionality

**Comment by mlautman** [comment]
@davetcoleman changes made.

File: doc/kinematic_model/kinematic_model_tutorial.rst [hunk 1]
```diff
@@ -1,40 +1,34 @@
 Kinematic Model Tutorial
 ========================
+In this section, we will walk you through the C++ API for using kinematics in MoveIt!.
 
-In this section, we will walk you through the C++ API for using kinematics.
+.. image:: panda_tf.png
+   :width: 500px
+   :alt: "That is a sweet robot" you might say.
```
Comment (davetcoleman) [inline doc/kinematic_model/kinematic_model_tutorial.rst]: haha

**Pull request merged**